concrete/src/Page/Stack/Stack.php

```diff
@@ -44,6 +44,13 @@ public static function mapImportTextToType($type)
         }
     }
 
+    /**
+     * @param string $path
+     * @param string $version
+     * @param TreeInterface|null $siteTree
```

Comment:
What about using the fully qualified type names in phpdocs? I mean, for example in this line: `@param \Concrete\Core\Site\Tree\TreeInterface|null $siteTree`

That way:
1. when we read the code we don't have to look at the `use` lines at the top of the file
2. when we generate "fake" classes for IDE we copy the phpdoc but not the top "use" statements: this lead to wrong typehinting when we use class names as defined in the top "use" statements.

Reply:
Implemented in https://github.com/concrete5/concrete5/pull/7660/commits/728af7648ff24da8917c6df1f8ca692777e5d9ca